<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataImport</class>
 <widget class="QDialog" name="DataImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>280</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fcfcfc;</string>
  </property>
  <widget class="QLabel" name="labelSelectTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select table:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpenFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select csv file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOpenFile">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: #fcfcfc;</string>
   </property>
   <item>
    <property name="text">
     <string>Компании</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Контакты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Инфо о компании</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Услуги</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Контракты</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonImport">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>240</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Imported</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;
color:  #1a1a1a;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinimize">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>-3</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;
color:  #1a1a1a;</string>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
